packages/verification-vue-component/src/style/components/v_tooltip.scss

```diff
@@ -7,7 +7,7 @@
         background-color: #052b50;
         border-radius: 3px;
         padding: 8px 12px;
-        color: white;
+        color: $white;
```

Comment:
This variable is not available here, you would have to import the variables file here too, like in the certifaction_verification file

Reply:
fixed it